examples/06_ken_embeddings.py

```diff
@@ -1,86 +1,101 @@
-capture and summarize the information on an entity.
-Relational data embeddings capture all common entities from Wikipedia. [#]_
-These will be called `KEN embeddings` in the following example.
+When the data contains common entities (cities, countries, companies,
+famous people, etc.), bringing new information assembled from external sources
+is a key to significantly improve the statistical analysis.
+
+Usually, this would be done by injecting our dataset with other tables,
+which we can for example download from the Internet.
+
+In this category, Wikipedia is one of the richest sources of information when
+dealing with these common entities. However, instead of downloading and joining
+massive tables extracted from Wikipedia, we can instead use **embeddings**.
+
+Embeddings, or vectorial representations of entities, are a convenient way to
+capture and summarize the information related to an entity.
+We will use `KEN embeddings`, which are embeddings extracted from Wikipedia
+(`YAGO <https://yago-knowledge.org/>_`). [#]_
+
+We will see how to use them in a tabular learning setting, and see whether
+they improve our model's accuracy (spoiler: they do).
 
-We will see that these embeddings of common entities significantly
-improve our results.
 
 .. note::
     This example requires `pyarrow` to be installed.
 
 .. [#] https://soda-inria.github.io/ken_embeddings/
 
+.. |Pipeline| replace::
+    :class:`~sklearn.pipeline.Pipeline`
+
+.. |OneHotEncoder| replace::
+    :class:`~sklearn.preprocessing.OneHotEncoder`
 
- .. |Pipeline| replace::
-     :class:`~sklearn.pipeline.Pipeline`
+.. |ColumnTransformer| replace::
+    :class:`~sklearn.compose.ColumnTransformer`
 
- .. |OneHotEncoder| replace::
-     :class:`~sklearn.preprocessing.OneHotEncoder`
+.. |MinHashEncoder| replace::
+    :class:`~skrub.MinHashEncoder`
 
- .. |ColumnTransformer| replace::
-     :class:`~sklearn.compose.ColumnTransformer`
+.. |get_ken_embeddings| replace::
+    :class:`~skrub.datasets.get_ken_embeddings`
 
- .. |MinHash| replace::
-     :class:`~skrub.MinHashEncoder`
+.. |HGBR| replace::
+    :class:`~sklearn.ensemble.HistGradientBoostingRegressor`
 
- .. |HGBR| replace::
-     :class:`~sklearn.ensemble.HistGradientBoostingRegressor`
+.. |FeatureAugmenter| replace::
+    :class:`~skrub.FeatureAugmenter`
 """
 
 ###############################################################################
-# The data
-# --------
+# Predicting video game sales
+# ---------------------------
 #
-# We will take a look at the video game sales dataset.
-# Let's retrieve the dataset:
+# In this example, we will take a look at a video game sales dataset.
+# We first retrieve our base dataset:
+
 import pandas as pd
 
 X = pd.read_csv(
     "https://raw.githubusercontent.com/William2064888/vgsales.csv/main/vgsales.csv",
     sep=";",
     on_bad_lines="skip",
 )
-# Shuffle the data
-X = X.sample(frac=1, random_state=11, ignore_index=True)
```

Comment:
This must stay because the data is ordered by video game sales. If you do not do this, you are creating a pattern that biases your prediction model.